pom.xml

```diff
@@ -180,8 +180,6 @@
              interferes with that. Use 'https' for builds that will be released
              to non-snapshot public maven repos -->
         <maven.repository.protocol>https</maven.repository.protocol>
```

Comment:
This should be deleted as well.

Reply:
This is needed to activate the `http` profile by setting value `-Dmaven.repository.protocol=http` to use `HTTP` repository instead of `HTTPS` one, I will explain in the below review comment.